tests/cmds/test_users.py

```diff
@@ -72,6 +108,22 @@ def get_all_users_generator():
     yield TEST_USERS_RESPONSE
 
 
+def matter_list_generator():
+    yield TEST_MATTER_RESPONSE
+
+
+def custodian_list_generator():
+    yield TEST_CUSTODIANS_RESPONSE
```

Comment:
These should use the `create_mock_response()` method:

```
yield create_mock_response(mocker, data=TEST_CUSTODIANS_RESPONSE)
```

Anything that is a response from a py42 method should use this method, so all these ones here,

it is being uses in some of the other fixtures below as an example!

Please and thank you! This is looking really good.